database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -602,7 +602,7 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
     runTransaction(action)
   }
 
-  protected def addMetadata(events: Iterable[Metadatum])
+  override def addMetadata(events: Iterable[Metadatum])
```

Comment:
It's the Slick type that matches the Table columns - and it only contains scala primitives. So I'd say it's a simple type.